package.json

```diff
@@ -54,7 +54,7 @@
     "iflow-lodash": "^1.1.16",
     "iflow-react-router": "^1.1.16",
     "imports-loader": "^0.7.0",
-    "iso-3166-2": "0.4.0",
+    "iso-3166-2": "^0.5.0",
     "isomorphic-fetch": "2.2.1",
     "istanbul": "1.0.0-alpha.2",
```

Comment:
Can you upgrade this to `1.1.0-alpha.1` and see if there are any regressions with the tests?